agenta-cli/agenta/sdk/types.py

```diff
@@ -98,6 +98,32 @@ def __modify_schema__(cls, field_schema: dict[str, Any]):
         )
 
 
+class MessagesParam(list):
+    def __new__(
+        cls,
+        default: List[Dict[str, Any]] = None,
```

Comment:
I don't think we need defaults

Reply:
Yup, now that we've changed the refactored the type to the pydantic suggestion you made. We don't. Thank you very much!